erts/emulator/beam/utils.c

```diff
@@ -3152,6 +3152,9 @@ Sint erts_cmp_compound(Eterm a, Eterm b, int exact, int eq_only)
 		    int cmp;
 		    byte* a_ptr;
 		    byte* b_ptr;
+            if (eq_only && a_size != b_size) {
+                RETURN_NEQ(a_size - b_size);
+            }
```

Comment:
Thanks for trying, but I don't think you got the indentation right.
Tabs should be 8 spaces (if used).
Identation is usually 4 spaces, which it is in this code.

While at it, change `RETURN_NEQ();` to `goto not_equal;`

Reply:
Thanks. The indentation should now match the surrounding code (two 8-wide tabs, followed by blocks of 4 spaces).